static/app/components/codecov/datePicker/dateSelector.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import {ConfigFixture} from 'sentry-fixture/config';
+import {UserFixture} from 'sentry-fixture/user';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {DateSelector} from 'sentry/components/codecov/datePicker/dateSelector';
+import ConfigStore from 'sentry/stores/configStore';
+
+const {router} = initializeOrg({
+  router: {
+    location: {
+      pathname: '/codecov/tests/',
+      query: {},
+    },
+    params: {},
+  },
+});
+
+describe('DateSelector', function () {
+  const onChange = jest.fn();
+
+  function getComponent(props = {}) {
+    return <DateSelector onChange={onChange} {...props} />;
+  }
+
+  function renderComponent(props = {}) {
+    return render(getComponent(props), {router});
+  }
```

Comment:
the types here are `{}`. Putting `<DateSelector onChange={onChange} />` in each test saves pain later